### ISSUE TITLE
Add Golem to Packagemanagers to ease the Installation Process and make it more public

### ISSUE THREAD

**Issue by tomwwabo** [golemfactory/clay#4931]
## Rationale

To gain a wider UserBase it would be great if, Golem would become part of package-managers

## Description

I have already created a Package Request Issue 
https://github.com/chocolatey-community/chocolatey-package-requests/issues/717

It would be enough to just upvote it.

would also be great if golem would become part of the following package managers

* http://chocolatey.org/
  * a widely adopted package manager for windows
* http://debops.org/
  * it is very important that golem becomes part of a
    * using galaxy.ansible.com 
* https://wiki.debian.org/DebianRepository
   * for debian users
* https://launchpad.net/
  * for ubuntu users
* https://flathub.org/home
  * for all linux users
* https://snapcraft.io/
   * basically cannonicals https://flatpak.org/, but the adoption of flatpak is greaterthan https://snapcraft.io/

Also please take a look at the overall marketing strategy over at [my comment on r/GolemProject](https://www.reddit.com/r/GolemProject/comments/dwue43/charity_boinclike_cloud_computing_interesting/f8esx0v?utm_source=share&utm_medium=web2x)

## Dependencies

* https://chocolatey.org/packages/docker-toolbox

Thx in advance
